client/my-sites/checkout/utils.ts

```diff
@@ -135,3 +144,49 @@ export function getAffiliateCouponLabel(): string {
 	// translators: The label of the coupon line item in checkout
 	return translate( 'Exclusive Offer Applied' );
 }
+
+export function useGetWpcomPlanTotalIfPaidMonthly( products: Array< ResponseCartProduct > ): {
+	[ key: string ]: number | undefined;
+} {
+	const plansToMonthly = products
+		.filter( ( product ) => {
+			return isWpComPlan( product?.product_slug ) && ! isMonthly( product?.product_slug );
+		} )
+		.map( ( product ) => {
+			return {
+				plan: product.product_slug,
+				monthly_plan: getMonthlyPlanByYearly( product.product_slug ),
```

Comment:
Do you need this intermediate structure? I don't think we do - but if we do, then I would name things this way:

```suggestion
				planSlug: product.product_slug,
				monthlyVariantPlanSlug: getMonthlyPlanByYearly( product.product_slug ),
```

Reply:
I've replaced it with a plain array.